### ISSUE TITLE
Add more explainer text to FAQ page

### ISSUE THREAD

**Issue by joenicolette** [loveology/design#22]
“Where’s my money going to? Towards the experts? Towards the website?”
As a not-for-profit, the volunteers and paid team of contractors at Loveology are as efficient and effective as possible with every dollar we receive from generous people like you. Our content experts are not paid. We are so grateful to them for donating their time and expertise.
Though Loveology does not charge for any service to users, we do rely on financial support to cover third party expenses that include things like:
Cloud Hosting
Video Production and Editing
Data Science
Project Management
SEO and Paid Advertising to reach people 
Closed Captioning
Customer Service
In short, all money raised for Loveology is used to maintain and develop the functionality of the site and make this free resource available everywhere.
As long as Loveology has support for these expenses we will continue to provide the service. 
Thank you for your interest. As always, we welcome your feedback and further questions.

<img width="1551" alt="Screen Shot 2021-12-10 at 4 47 32 PM" src="https://user-images.githubusercontent.com/82427400/145645419-24055b59-6016-4067-a411-8422371c979d.png">

**Comment by gitethanwoo** [comment]
Clarification - is this asking to add this text to the donate page?

**Comment by joenicolette** [comment]
*user testing voice* What do _you_ think it means?

**Comment by joenicolette** [comment]
Yes @gitethanwoo

**Comment by Angelinagwoo** [comment]
Here are the designs I had in mind, they have the same layout as the FAQ section of the site:
<img width="477" alt="Screen Shot 2022-02-24 at 1 38 26 PM" src="https://user-images.githubusercontent.com/96139630/155587018-4eef5f8c-8513-4291-b9df-c651098f1991.png">

Text here:
<img width="754" alt="Screen Shot 2022-02-24 at 1 38 33 PM" src="https://user-images.githubusercontent.com/96139630/155587087-95304667-1424-4a5f-8b08-ebae2a88e853.png">

Link to complete designs, including mobile, here: https://www.figma.com/file/wd0rtbT9KgvtsTZj3e36im/Archived-Designs?node-id=7%3A42302
